.gitignore

```diff
@@ -1 +1,4 @@
-vendor
\ No newline at end of file
+vendor
+/.idea/
```

Comment:
IDE/Editor related ignores should be handled globally in your git config, not per project